src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -1066,23 +1114,25 @@ export const gravityStitchingEnvironment = (
             })
           },
         },
-        marketingCollections: {
-          fragment: gql`
+        ...(!useUnstitchedMarketingCollections && {
+          marketingCollections: {
+            fragment: gql`
               ...on Viewer {
                 __typename
               }
             `,
-          resolve: async (_parent, args, context, info) => {
-            return await info.mergeInfo.delegateToSchema({
-              schema: gravitySchema,
-              operation: "query",
-              fieldName: "marketingCollections",
-              args,
-              context,
-              info,
-            })
+            resolve: async (_parent, args, context, info) => {
+              return await info.mergeInfo.delegateToSchema({
+                schema: gravitySchema,
+                operation: "query",
+                fieldName: "marketingCollections",
+                args,
+                context,
+                info,
+              })
+            },
           },
-        },
+        }),
```

Comment:
Yep - lots of changes in this file - conditionally merging in type defs and resolvers based on the config - and oof marketing collections was definitely stitched in a solid number of places!

Nevertheless this is what I expected to see - more or less - open to ideas on how to make it easier for future migrations!